vernac/vernacstate.ml

```diff
@@ -71,3 +71,70 @@ let make_shallow st =
     system = States.replace_lib st.system @@ Lib.drop_objects lib;
     shallow = true;
   }
+
+(* Compatibility module *)
+module Proof_global = struct
+
+  let with_fail ~st f = f ()
```

Comment:
What's this? It explains the `paralleltac.v` failure, I believe...

Reply:
Ouch, what should the code be here?

Reply:
What it was before, I guess, was there a problem with the existing code in vernacentries?

Reply:
That code has always been a bit strange, indeed let me try to connect it.

Reply:
Oh the reason is one of dependencies!

Reply:
Pushed a commit let's see ...